Gulpfile.js

```diff
@@ -149,11 +149,12 @@ gulp.task('lint-client', function () {
             'src/client/runner/automation/playback/drag.js',
             'src/client/runner/automation/playback/rclick.js',
             'src/client/runner/automation/playback/hover.js',
+            'src/client/runner/automation/playback/click/**/*.js',
             'src/client/runner/automation/cursor.js',
             'src/client/runner/automation/automation.js',
-            'src/client/runner/automation/util.js',
             'src/client/runner/utils/**/*.js',
-            'src/client/ui/cursor'
+            'src/client/ui/cursor',
+            'src/client/runner/automation/utils.js'
```

Comment:
let's keep some grouping and place `src/client/ui/..` after all `src/client/runner/...` as it was before